# Request a plan visualization. If no plan available, always fails

---
bool success
